styles.css

```diff
@@ -0,0 +1,686 @@
+  line-height: 14px;
+  color: var(--black);
+}
+
+.speakerbtn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 4px;
+  width: 70%;
+  max-width: 500px;
+  justify-content: center;
+  margin: auto;
+  transition: 300ms ease ease-out;
+}
+
+.speakerbtn:hover {
+  cursor: pointer;
+}
+
+.arrowdown {
+  width: 8px;
+  height: 5.33px;
+  order: 1;
+  flex-grow: 0;
+}
+
+/* about html styles */
+.margintop {
+  padding-top: 4rem;
+}
+
+.center {
+  text-align: center;
+}
+
+.email_contact > p {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 25px;
+  align-items: center;
+  text-align: center;
+  color: #7a7a7a;
+  width: 80%;
+  margin: auto;
+}
+
+.contactEmail {
+  font-style: normal;
+  font-weight: 800;
+  font-size: 12px;
+  margin: auto;
+  line-height: 25px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.02em;
+  text-decoration-line: underline;
+  color: #5f5f5f;
+  padding-bottom: 2rem;
+}
+
+.summits_info {
+  background: var(--white);
+}
+
+.logoInfo h3 {
+  text-align: center;
+  padding-top: 1.2rem;
+}
+
+.logoInfo p {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 21px;
+  padding: 4px 2px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: -0.03em;
+  color: #838383;
+}
+
+.summit_logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+/* prev summit */
+.previpusSummit h3 {
+  padding: 2rem 1.2rem;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  letter-spacing: -0.02em;
+  color: #2c2c2c;
+}
+
+.previpusSummit p {
+  padding: 0 0 1rem 0;
+  text-align: center;
+  font-weight: 700;
+  font-size: 12px;
+  width: 80%;
+  margin: auto;
+  line-height: 21px;
+  align-items: center;
+  color: #838383;
+}
+
+.prevSummit {
+  display: flex;
+  flex-direction: column;
+  gap: 1.3rem;
+  padding: 1.3rem;
+  justify-content: center;
+}
+
+.partners {
+  align-items: center;
+  padding: 20px 0 40px;
+  gap: 10px;
+  background: #414246;
+}
+
+.partners h5 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+  letter-spacing: -0.01em;
+  color: var(--white);
+}
+
+.patnerslogo {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.patnerslogo img {
+  padding: 0.3rem;
+}
+
+/* footer */
+.aboutfooter {
+  background-color: var(--white);
+  color: var(--black);
+}
+
+.footer {
+  display: grid;
+  grid-template-columns: auto auto;
+  padding: 1.2rem;
+  gap: 1rem;
+}
+
+.aboutfooter img {
+  width: 150px;
+}
+
+.aboutfooter p {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 10px;
+  line-height: 13px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.01em;
+  color: #3b3e45;
+}
+
+.aboutFooter {
+  background-color: #3e3e3e;
+  padding: 3rem 1rem;
+  color: var(--white);
+}
+
+.white {
+  color: var(--white) !important;
+}
+
```

Comment:
- [ ] Good job implementing the requested changes for About's footer but in the mobile menu now the text is invisible. The background is white and the text is in white also, that is why we can't see the text, the text should be in black color. Kindly fix that 🙏🏾 

<img width="437" alt="Screenshot 2023-08-01 at 15 44 34" src="https://github.com/MPDADDY/capstone_prize-giving_day/assets/74501523/53566ca0-1494-4480-97b5-2884ccbe3173">